src/CopyrighFixer/CopyrighFixer/worker.h

```diff
@@ -32,6 +38,10 @@ class CopyrighFixer_EXPORT Worker {
      */
     void printHelp() const;
 
+protected:
+    Config *conf_;
+    ConfigParser *confParser_;
+    Signer *subscriber_;
```

Comment:
the name of private value must be begin from a '_' char 
```suggestion
    Config *_conf;
    ConfigParser *_confParser;
    Signer *_subscriber;
```